test/docker/Dockerfile

```diff
@@ -35,12 +35,12 @@ RUN chmod +x /usr/local/bin/repo
 
 # create istioctl from `master`, assuming that's what the builder had locally
 # ignores `The command '/bin/sh -c go get -d istio.io/istio' returned a non-zero code: 1`
-RUN go get -d istio.io/istio || true   
+RUN git clone https://github.com/istio/istio /go/src/istio.io/istio
 RUN cd /go/src/istio.io/istio && make istioctl
 
 ############################################
 # Main image
-FROM  kindest/node:v1.14.1
+FROM kindest/node:v1.15.0@sha256:b4d092fd2b507843dd096fe6c85d06a27a0cbd740a0b32a880fe61aba24bb478
```

Comment:
They don't have a tag ?

Reply:
`v1.15.0` is the tag. In the release notes the recommend adding the sha I guess its not needed though

Reply:
I really don't like SHAs copied by hand. It's the job of tools.

Reply:
I just removed it